### PR TITLE
add aat config for cdam

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-2294

### Change description ###
add aat config for cdam

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/ccd/ccd-case-document-am-api/aat.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
```
Comment (danlysiak) [inline k8s/namespaces/ccd/ccd-case-document-am-api/aat.yaml:1]: Need to add this patch to kustomize too:
https://github.com/hmcts/cnp-flux-config/blob/9996a0513e6ec0b73ddd360575ab89b9fe7122a4/k8s/aat/common-overlay/ccd/kustomization.yaml#L10

**Review by kaanaktas — DISMISSED** [review]
Lgtm

**Review by kaanaktas — APPROVED** [review]
Lgtm

**Review by kaanaktas — APPROVED** [review]
Lgtm

**Review by hemantt — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**